### PR TITLE
fix: hotfixes relayout

### PR DESCRIPTION
**Pull request by rebelchris**
Fixed a placement issue of the mobile article page not being full width
Tooltip minimized hover was not covering the full space

### CODE REVIEW

File: packages/webapp/pages/posts/[id].tsx [hunk 1]
```diff
@@ -238,7 +238,7 @@ const PostPage = ({ id, postData }: Props): ReactElement => {
 
   return (
     <>
-      <div className="flex relative flex-col flex-wrap mx-auto max-w-full">
+      <div className="flex relative flex-col flex-wrap mx-auto w-full laptopL:w-auto max-w-full">
```
Comment (idoshamun) [inline packages/webapp/pages/posts/[id].tsx:241]: Why the `laptopL:auto`? We already set the width for this resolution in the PageContainer element

**Review by idoshamun — CHANGES_REQUESTED** [review]
Looks great just one question

File: packages/webapp/pages/posts/[id].tsx (hunk 1 above)
Comment (rebelchris) [reply packages/webapp/pages/posts/[id].tsx:241]: weird case where the full class would overwrite the one we set.
Auto would just take that one and unset the full width.

Comment (idoshamun) [reply packages/webapp/pages/posts/[id].tsx:241]: Ok!

**Review by idoshamun — APPROVED** [review]

**Pull request merged**